testing/test_pytester.py

```diff
@@ -443,7 +443,7 @@ def test_one():
 
 
 def test_unicode_args(testdir) -> None:
-    result = testdir.runpytest("-k", "💩")
```

Comment:
Turns out a pile of poo is not a valid identifier -- this passed previously simply because this doesn't have any tests so wasn't parsed at all.